### DISCUSSION TITLE
Q: Appimage naming convention

### DISCUSSION THREAD

**Discussion by efb4f5ff-1298-471a-8973-3d47447115dc** [AppImage/AppImageKit#1375] [Q&A] [answered]
Hi Im wondering if there is a certain naming convention for Appimages. 

The only things i could find in the docs:
- https://docs.appimage.org/packaging-guide/distribution.html#do-not-put-linux-into-the-appimage-file-name
- https://docs.appimage.org/packaging-guide/distribution.html#do-not-put-appimages-into-other-archives

On the legacy forum i could find 1 answer from @probonopd:
- https://discourse.appimage.org/t/portable-configuration-data/105/24

I assume this is the correct answer. Followup q with that assumption in mind, is `ApplicationName_1.0.0_arm64.AppImage` the correct way of naming package?

**Comment by probonopd** [comment, marked as answer]
`ApplicationName-1.0.0-aarch64.AppImage` or `Application_Name-1.0.0-aarch64.AppImage`. (Note `-` between the parts; `_` is used if the application name contains a blank.)
But if you are using one of the higher-level tools, then it will choose the filename automatically for you, typically using information coming from the bundled desktop file.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Thank you so much for you reply. Have a nice day!

**Comment by Samueru-sama** [comment]
Note that `ApplicationName-$VERSION-$ARCH.AppImage` is actually documented, the issue is that it is on the draft spec: https://github.com/AppImage/AppImageSpec/blob/master/draft.md#type-2-image-format
